installation/scripts/is-installed.sh

```diff
@@ -33,18 +86,29 @@ do
       break
   elif [ "$STATUS" = "Error" ]
   then
-    echo "kyma installation error... ${DESC}"
-    echo "----------"
-    echo "$(kubectl logs -n kyma-installer $(kubectl get pods --all-namespaces -l name=kyma-installer --no-headers -o jsonpath='{.items[*].metadata.name}'))"
-    echo "----------"
-    exit 1
+    echo "kyma installation error, description: ${DESC}"
+    if [ "$TIMEOUT_SET" -eq 0 ]; then
+      echo "to fetch the logs from the installer execute: kubectl logs -n kyma-installer $(kubectl get pods --all-namespaces -l name=kyma-installer --no-headers -o jsonpath='{.items[*].metadata.name}')"
+    fi
+    if [ "$TIMEOUT_SET" -ne 0 ] && [ "$ITERATIONS_LEFT" -le 0 ]; then
+      echo "timeout reached on kyma installation error. Fetching logs from the installer:"
+      echo "----------"
+      echo "$(kubectl logs -n kyma-installer $(kubectl get pods --all-namespaces -l name=kyma-installer --no-headers -o jsonpath='{.items[*].metadata.name}'))"
+      echo "----------"
+      exit 1
+    fi
   else 
     echo "Status: ${STATUS}, description: ${DESC}"
-    if [ "${VERBOSE}" -eq 1 ]; then
-      echo "$(kubectl get installation/kyma-installation -o yaml)"
-      echo "----------"
-      echo "$(kubectl get po --all-namespaces)"
+    if [ "$TIMEOUT_SET" -ne 0 ] && [ "$ITERATIONS_LEFT" -le 0 ]; then
+      echo "timeout reached. Exiting"
+      exit 1
     fi
-    sleep $DELAY
   fi
+  if [ "${VERBOSE}" -eq 1 ]; then
+    echo "$(kubectl get installation/kyma-installation -o yaml)"
```

Comment:
Is there a particular reason to use command substitution / subshell here? It's more explicit to just run the command directly.

Reply:
The same answer as in your previous comment.